app/controllers/zip_code_controller.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class ZipCodeController < ApplicationController
+  # GET '/zipcodes'
+  def index
+    render json: ZipCode.all.take(50)
```

Comment:
Once you paginate this, you don't need the `take`, in the future if you need to limit the number of records, you can use the `limit` method, such as:

```ruby
ZipCode.all.limit(50)
```

This way is better because you just fetch 50 instead of first fetching the 40K records and then just taking 50 through ruby

Reply:
Done!